app/js/components/drugOrderEntry/addForm/AddForm.jsx

```diff
@@ -149,7 +146,7 @@ export class AddForm extends React.Component {
         item = drugDispensingUnits.filter(unit => unit.display === value).length > 0;
         break;
       default:
-        item = value.length > 0;
+        // does nothing
```

Comment:
Can you elaborate and through more colour.

Reply:
That particular line of code `item = value.length > 0;` does nothing, there is actually no switch value that would result in that default. You can confirm that form the way the `handleValidation` method is used in `FreeText.jsx` and `StandardDose.jsx`

The reason I removed it is because it is dropping the code coverage of the project.